File: PessoaPython/pessoa.py
# CLASSE Pessoa
class Pessoa:

# Atributos da Classe Pessoa
    def __init__(self, nome, sexo, idade, altura):
        self.nome = nome
        self.sexo = sexo
        self.idade = idade
        self.altura = altura

    # Métodos da Classe Pessoa
    def Feliz(self):
        print('Estado: feliz!')

    def Triste(self):
        print('Estado: triste!')

    def Doente(self):
        print('Estado: doente!')
    
    def Choro(self):
        print('Estado: chorando!')

    def Raiva(self):
        print('Estado: raiva!')

    def ExibirPessoa(self):
        print('Pessoa:', self.nome, self.sexo, self.idade, self.altura)

File: PessoaJava/Pessoa.java
// Classe Pessoa
//package classe;

public class Pessoa {
    // Atributos da Classe Pessoa
    public String nome;
    public String sexo;
    public String idade;
    public String altura;

    // Métodos da Classe Pessoa
    public void feliz(){
        System.out.println("Estado: feliz!\n");
    }

    public void triste(){
        System.out.print("Estado: triste!\n");
    }

    public void doente(){
    System.out.print("Estado: doente!\n");
    }

    public void choro(){
        System.out.println("Estado: choro!\n");
    }

    public void raiva(){
        System.out.println("Estado: raiva!\n");
    }

    public void exibirpessoa(){
        System.out.println("Pessoa: " + nome + " " + sexo + " " + idade + " " + altura);
    }
}
